tools/tools.go

```diff
@@ -32,4 +32,5 @@ import (
 	_ "k8s.io/code-generator/cmd/openapi-gen"
 	_ "k8s.io/code-generator/cmd/register-gen"
 	_ "k8s.io/code-generator/cmd/set-gen"
+	_ "github.com/kubernetes-incubator/apiserver-builder-alpha/cmd/apiregister-gen"
```

Comment:
why not use `sigs.k8s.io/apiserver-builder-alpha/cmd/apiregister-gen`?

this process is moved to https://github.com/kubernetes-sigs/apiserver-builder-alpha/  now.

Reply:
The pinned version v1.12 was still using the `kubernetes-incubator` import path, we can use `sigs.k8s.io....` after we bump the kubernetes dependencies to `v1.16`